src/app/legacy/containers/CpsRecommendations/RecommendationsPromoList/index.test.jsx

```diff
@@ -31,10 +30,21 @@ const Fixture = () => (
 beforeEach(jest.clearAllMocks);
 
 describe('RecommendationsPromoList', () => {
-  shouldMatchSnapshot(
-    'it renders a list of Story Promos wrapped in Grid components',
-    <Fixture />,
```

Comment:
Maybe this is where the comma came from?
```
<Fixture />,
```

when it should just have been
```
<Fixture />
```

Reply:
Good thinking! But I don't think so... 🙁 On latest, I get a linting error if I try to remove the comma, and it seems to be the standard syntax for the shouldMatchSnapshot() function. E.g.

shouldMatchSnapshot(
    'blah blah',
    <Component />,
  );

Reply:
OK. And when you remove that comma, does it cause a linting issue? Either way, it looks like it's just the test code that's affected, not the "real" code i.e. we're not accidentally removing a comma from the website when it's supposed to be there. So I think this is OK, and happy to merge as is.

Reply:
Great, yep I agree. 👍

(I did actually get to the bottom of it in the PR - it's the comma at the [end of this line](https://github.com/bbc/simorgh/blob/011ee711556e34c21fe37516adab7ef38312b10f/src/app/legacy/containers/CpsRecommendations/RecommendationsPromoList/index.test.jsx#L25). Good to know!)

Reply:
Awesome, thanks for investigating and sharing here!